pom.xml

```diff
@@ -890,7 +943,16 @@
                 </plugins>
             </build>
             <modules>
+                <module>extensions</module>
```

Comment:
Profiles `release` , `release-snapshot` and `zip` only listed the `hazelcast-all` module in their respective `modules` sections. I am not sure if the intent was to only build `hazelcast-all`, however effectively with the previous pom.xml state these profiles would always build all modules, that's why I included all modules here.